test.py

```diff
@@ -16,11 +16,97 @@
 parser.add_argument('--config_path', type=str, default='config/nusc_config.yaml')
 parser.add_argument('--detection_path', type=str, default='data/detector/val/val_centerpoint_new.json')
 parser.add_argument('--first_token_path', type=str, default='data/utils/first_token_table/trainval/nusc_first_token.json')
+parser.add_argument('--end_token_path', type=str, default='data/utils/end_token_table/trainval/nusc_first_token.json')

```

Comment:
Rename `nusc_first_token.json` to `nusc_end_token.json`, and please provide the `nusc_end_token.json` file along with the obtained script.